tetrateci/eks-ingress.1.8.patch

```diff
@@ -1,3 +1,21 @@
+diff --git a/pkg/test/framework/components/echo/kube/instance.go b/pkg/test/framework/components/echo/kube/instance.go
+index 9b320a8e32..b2d9098218 100644
+--- a/pkg/test/framework/components/echo/kube/instance.go
++++ b/pkg/test/framework/components/echo/kube/instance.go
+@@ -220,10 +220,9 @@ func createVMConfig(ctx resource.Context, c *instance, cfg echo.Config) error {
+ 		if cfg.AutoRegisterVM {
+ 			cmd = append(cmd, "--autoregister")
+ 		}
+-		if !ctx.Environment().(*kube.Environment).Settings().LoadBalancerSupported {
```

Comment:
Do we have to remove this? I think it doesn't matter

Reply:
Meaning, if we don't have to remove it, lets keep that if block.

Reply:
It doesn't matter we can do it either way

Reply:
I would retain it as it means less difference from the original source.